docs/user/alternatives.rst

```diff
@@ -13,7 +13,7 @@ When deciding which library is most useful to you, it is worth considering the f
 
    Software is a living, breathing organism and is constantly undergoing change. If any of the above information is incorrect or out of date, or if you want to add a new project to this list, please open a PR!
 
-   *Last updated: 31 March 2025*
+   *Last updated: 15 April 2025*
```

Comment:
Let's keep this date the same, unless you also want to update all of the other libraries too. Luckily it looks like not much has changed in GeoDeep in the last couple weeks.